### ISSUE TITLE
Timestamps are copied along with logs in Firefox and Safari

### ISSUE THREAD

**Issue by jama22** [concourse/concourse#4060]
Opening this issue as a result of discussion in #2066 . Repro'ed on 5.3.0 on prod

# Steps to Reproduce
1. Launch Firefox / Safari
2. Go to a build page on prod like https://ci.concourse-ci.org/teams/main/pipelines/concourse/jobs/upgrade/builds/401
3. Try to copy some build output
4. Paste it into a notepad

# Expected Results
I would expect to only see the build output without the timestamps. Try doing this with Chrome

# Actual Results
The timestamps are copied along with the build logs

**Comment by univ0298** [comment]
This issue continues to exist in v5.4.1

**Comment by vito** [comment]
Yeah this is pretty annoying. It seems to depend on how you select the text. I'll chuck it in the side-road backlog.

**Comment by taylorsilva** [comment]
Just tried reproducing this in Firefox and I can't anymore! It fixed itself!
